src/core/HorizontalTabs/HorizontalTabs.tsx

```diff
@@ -161,30 +216,32 @@ export const HorizontalTabs = (props: HorizontalTabsProps) => {
         )}
         role='tablist'
         ref={refs}
+        onKeyDown={onKeyDown}
         {...rest}
       >
         {labels.map((label, index) => {
           const onClick = () => onTabClick(index);
           return (
             <li key={index}>
-              {typeof label === 'string' ? (
+              {!React.isValidElement(label) ? (
```

Comment:
how this check changes/helps than before?

Reply:
It is a typeguard (implemented using `is` keyword), so there is no need to cast to `JSX.Element` anymore.

Reply:
Honestly, I think we should always check for this before cloning (in all places)